spring-batch-infrastructure/src/main/java/org/springframework/batch/item/database/AbstractCursorItemReader.java

```diff
@@ -485,7 +485,7 @@ protected T doRead() throws Exception {
 
 	/**
 	 * Read the cursor and map to the type of object this reader should return. This method must be
-	 * overriden by subclasses.
+	 * override by subclasses.
```

Comment:
This should be "overridden". I can do it while rebasing this PR.

Reply:
ok. I fixed it.